src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -69,6 +70,10 @@ private void configureComponents() {
     shooterRight.setStatusFramePeriod(StatusFrame.Status_1_General, 255);
     shooterRight.setStatusFramePeriod(StatusFrame.Status_2_Feedback0, 255);
 
+    if (RobotConfig.ROBOT_IS_COMPETITION) {
+      shooterRight.setInverted(true);
```

Comment:
Same comment as above - why are we only inverting the comp bot motor and not both?

Reply:
Comp bot has motors facing opposite of each other - different than the practice bot.